independent-publisher-connectors/De Lijn/readme.md

```diff
@@ -0,0 +1,45 @@
+# De Lijn
+'De Lijn' is the public transport company for busses in Flanders. This is a part of Belgium.
+This service allows you to find a stop/line based on specific terms or location (lat/longitude)
+
+The api that is used is published by 'De Lijn' - I am not the owner of this endpoint and a Power Automate enthousiast.
```

Comment:
@Lschockaert all files should be free of grammatical errors and typos. Please change the following
"enthousiast" to "enthusiast"